src/main/java/org/truffleruby/parser/BodyTranslator.java

```diff
@@ -273,15 +274,13 @@
 import org.truffleruby.platform.graal.AssertConstantNodeGen;
 import org.truffleruby.platform.graal.AssertNotCompiledNodeGen;
 
-import java.math.BigInteger;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Deque;
-import java.util.Iterator;
-import java.util.List;
+import com.oracle.truffle.api.CallTarget;
+import com.oracle.truffle.api.Truffle;
+import com.oracle.truffle.api.frame.FrameSlot;
+import com.oracle.truffle.api.nodes.NodeUtil;
+import com.oracle.truffle.api.object.DynamicObject;
+import com.oracle.truffle.api.source.Source;
+import com.oracle.truffle.api.source.SourceSection;
```

Comment:
Changing the `import` order is annoying when rebasing, try to avoid as possible.